src/components/modules/banners/contributions/ContributionsBannerSecondaryCta.tsx

```diff
@@ -1,33 +1,58 @@
 import React from 'react';
 import { ThemeProvider } from '@emotion/react';
+import { css } from '@emotion/core';
 import { buttonReaderRevenueBrandAlt } from '@guardian/src-button';
 import { LinkButton } from '@guardian/src-button';
 import { SvgArrowRightStraight } from '@guardian/src-icons';
+import { space } from '@guardian/src-foundations';
+import { neutral } from '@guardian/src-foundations/palette';
+import { Button } from '@guardian/src-button';
+import { SecondaryCtaType } from '../../../../types/shared';
+import { ContributionsBannerEnrichedSecondaryCta } from './ContributionsBannerWrapper';
 
-interface ContributionsBannerSecondaryCtaProps {
-    onCtaClick: () => void;
-    ctaText: string;
-    ctaUrl: string;
+const reminderButtonStyles = css`
+    color: ${neutral[0]};
+    margin-left: ${space[4]}px;
+`;
+
+export interface ContributionsBannerSecondaryCtaProps {
+    secondaryCta: ContributionsBannerEnrichedSecondaryCta;
+    onCustomCtaClick: () => void;
+    onReminderCtaClick: () => void;
 }
 
 export const ContributionsBannerSecondaryCta: React.FC<ContributionsBannerSecondaryCtaProps> = ({
-    onCtaClick,
-    ctaText,
-    ctaUrl,
-}: ContributionsBannerSecondaryCtaProps) => {
+    secondaryCta,
+    onCustomCtaClick,
+    onReminderCtaClick,
+}) => {
     return (
-        <ThemeProvider theme={buttonReaderRevenueBrandAlt}>
-            <LinkButton
-                priority="tertiary"
-                size="small"
-                icon={<SvgArrowRightStraight />}
-                iconSide="right"
-                nudgeIcon={true}
-                onClick={onCtaClick}
-                href={ctaUrl}
-            >
-                {ctaText}
-            </LinkButton>
-        </ThemeProvider>
+        <>
+            {secondaryCta.type === SecondaryCtaType.Custom && (
+                <ThemeProvider theme={buttonReaderRevenueBrandAlt}>
+                    <LinkButton
+                        priority="tertiary"
+                        size="small"
+                        icon={<SvgArrowRightStraight />}
+                        iconSide="right"
+                        nudgeIcon={true}
+                        onClick={onCustomCtaClick}
+                        href={secondaryCta.cta.ctaUrl}
+                    >
+                        {secondaryCta.cta.ctaText}
+                    </LinkButton>
+                </ThemeProvider>
+            )}
+
+            {secondaryCta?.type === SecondaryCtaType.ContributionsReminder && (
```

Comment:
should this check if the reminder cookie exists?

Reply:
it should!

Reply:
okay that's updated now!

Reply:
👍  does the epic also have a `hasSetReminder`?

Reply:
just updated that to use this new function too 👍